homework-submission/week5/practice-app/src/components/login.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+class Login extends React.Component {
+  state = {
+    userName: '',
+    isLoggedIn: false,
```

Comment:
also what are using this variable for?
just if you click login with empty user name it will not going to HomeScreen, but it will change this variable ( isLoggedIn ) to true, so why....?